eslint.config.js

```diff
@@ -101,16 +95,20 @@ export default tseslint.config(
 			},
 		},
 	},
+	{
+		extends: [tseslint.configs.disableTypeChecked],
+		files: ["**/*.md/*.ts "],
+	},
 	{
 		extends: [vitest.configs.recommended],
 		files: ["**/*.test.*"],
 		rules: {
-			// These on-by-default rules aren't useful in test files.
```

Comment:
Removed intentionally. I don't think it was worth the space. This block is clearly for test files, and this list isn't something any project of mine has reasonably added to IIRC.